app/controllers/stations/dashboard_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Stations
+  class DashboardController < ApplicationController
+    def index
```

Comment:
I need to figure out how to get presenters working to get rid of all these instance variables. If you know of a better way in the interim then let me know and we can fix it.

Reply:
Hmm, I'm not too sure on presenters. The instance variables work for the time being